etc/freertos.mips4k.pic32mx.mk

```diff
@@ -55,11 +55,12 @@ ASFLAGS = -c -g -EL -MD -MP $(BASEDEFS) -D__LANGUAGE_ASSEMBLY__ -fdollars-in-ide
 #           -march=armv7-m -mthumb -mfloat-abi=soft
 
 CORECFLAGS = -c -EL -g -msoft-float -march=mips32r2 $(ARCHOPTIMIZATION) \
-	     -Wall -Werror -Wno-unknown-pragmas -MD -MP \
```

Comment:
Is there a specific warning that we cannot fix? -Werror contributes to codebase health.